figures/scripts/README.md

```diff
@@ -0,0 +1,11 @@
+## Scripts exclusively for publication-ready plots
+
+We list information about the relevant figures and modules for each script in this directory below for convenience.
+
+| Individual script | Figure(s) | Linked analysis modules |
+|--------|--------|------------------|
+| [`fig2-oncoprint-landscape.R`](fig2-oncoprint-landscape.R) | Figure 2 & Supplement | [`oncoprint-landscape`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/oncoprint-landscape) |
```

Comment:
Are these links to the scripts broken because they involve changes that haven't happened yet? 

Edit: I saw the note :+1:
> The documentation for the individual figure scripts reflects the changes in #1200